design/one-pager-stacks-security-isolation.md

```diff
@@ -69,10 +69,12 @@ This is described in greater details in the sections below.
 
 A few notes on the scope of access to CRDs:
 
-* All CRDs the stack owns and depends on are expected to be namespace scoped CRDs. Access to cluster scoped CRDs will not be granted, even for Cluster Stacks.
+* All CRDs a Namespaced stack owns must be namespace scoped CRDs. Cluster scoped Stacks may own cluster scoped CRDs.
```

Comment:
Thoughts on bumping revision version of this doc? Relatively significant change here so might be worth increment.

Reply:
Good thinking.  Updated with a revision history.